api/src/pdc/services/trip/actions/BuildExportAction.ts

```
@@ -91,325 +20,14 @@ export interface FlattenTripInterface extends ExportTripInterface<string> {
   ]],
 })
 export class BuildExportAction extends Action {
-  constructor(
-    private tripRepository: TripRepositoryProvider,
-    private fileProvider: S3StorageProvider,
-    private kernel: KernelInterfaceResolver,
-    private buildFile: BuildFile,
-  ) {
+  constructor() {
     super();
   }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)